driver.go

```diff
@@ -205,6 +213,10 @@ func (d *Driver) PreCreateCheck() error {
 		return errors.Wrap(err, "could not get location")
 	}
 
+	if d.UsePrivateNetwork == true && (d.networks == nil || len(d.networks) <= 0) {
```

Comment:
```suggestion
	if d.UsePrivateNetwork == true && (d.networks == nil || len(d.networks) == 0) {
```
the length can't be smaller than 0 afaik.